godot-kotlin/godot-library/build.gradle.kts

```diff
@@ -9,9 +11,17 @@ plugins {
 kotlin {
     // we don't have godot-library in the mobile targets yet, limit these to desktop for now
     //has to be change in `GodotPlugin` of `godot-gradle-plugin` as well
-    macosX64("macos")
-    linuxX64("linux")
-    mingwX64("windows")
+    if (platform != null) {
```

Comment:
As mentioned above this is problematic. What your doing here is saying that `godot-library` only supports `platform`. During publication, regardless of what OS you are building for, the targets should be configured for _all_ platform that we support. This is required to publish multiplatform libraries.

Reply:
right, that was causing trouble in former implem, I change it.